spec/text-buffer-spec.coffee

```diff
@@ -1049,6 +1050,22 @@ describe "TextBuffer", ->
         expect(bufferToDelete.getPath()).toBe filePath
         expect(bufferToDelete.isModified()).toBeFalsy()
 
+
+    describe "when the file is deleted", ->
+      events = []
+      beforeEach ->
+        bufferToDelete.onDidDelete -> events.push true
+        deleteHandler = jasmine.createSpy('deleteHandler')
+        bufferToDelete.onDidDelete deleteHandler
+        removeSync(filePath)
```

Comment:
Just curious, why is `removeSync` called here and also down below in the `it block?